### PR TITLE
Add links to publish request form

### LINKED ISSUE #167: Publish Request: Add Links
We need to add direct links on the publish request form to the License Agreement, the Statement of Faith, and the Translation Guidelines.

### PR DESCRIPTION
**Pull request by phillip-hopper**
Resolves #167

[Review on Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationdatabaseweb/411)

### CODE REVIEW

**Comment by phillip-hopper** [comment]
@vleong2332: Please review this.

File: td/publishing/forms.py [hunk 1]
```diff
@@ -130,11 +130,25 @@ class PublishRequestForm(forms.ModelForm):
...
+            'Statement of Faith': 'https://unfoldingword.org/faith',
+            'Translation Guidelines': 'https://unfoldingword.org/guidelines'
+        }
+
+        help_links = []
+
+        for key in links:
+            help_links.append('<a href="' + links[key] + '" target="_blank">View ' + key + '</a><br>')
+
+        self.fields["license_agreements"].help_text = '\r\n'.join(help_links)
```
Comment (vleong2332) [inline td/publishing/forms.py]: Clever use of help_text.

File: td/tests/publishing/test_forms.py [hunk 2]
```diff
@@ -59,6 +59,12 @@ def test_rejected_fields(self):
         form = PublishRequestNoAuthForm({}, None)
         self.assertFalse('rejected_at' in form.fields)
 
+    def test_license_links(self):
+        form = PublishRequestNoAuthForm({}, None)
+        self.assertTrue('license_agreements' in form.fields)
+        self.assertTrue('https://unfoldingword.org/license' in
```
Comment (vleong2332) [inline td/tests/publishing/test_forms.py]: Do we need to also check for the existence of `statement of faith` and `translation guidelines` being in form.fields.help_text?

Comment (phillip-hopper) [inline td/tests/publishing/test_forms.py]: That would probably be good. I'll stop being lazy and put those checks in also.

**Comment by vleong2332** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/unfoldingword-dev/translationdatabaseweb/411)*

**Pull request merged**